services/integration.sh

```diff
@@ -102,10 +103,27 @@ run cannon "2" ${orange}
 run cargohold "" ${purpleish}
 run spar "" ${orange}
 
+function run_nginz() {
+    colour=$1
+    prefix=$([ -w /usr/local ] && echo /usr/local || echo "${HOME}/.wire-dev")
+    (cd ${NGINZ_WORK_DIR} && LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${prefix}/lib/ ${TOP_LEVEL}/dist/nginx -p ${NGINZ_WORK_DIR} -c ${NGINZ_WORK_DIR}/conf/nginz/nginx.conf -g 'daemon off;' || kill_all) \
+        | sed -e "s/^/$(tput setaf ${colour})[nginz] /" -e "s/$/$(tput sgr0)/" &
+}
+
+NGINZ_PORT=""
+if [[ $INTEGRATION_USE_NGINZ -eq 1 ]]; then
+    NGINZ_PORT=8080
+    # Note: for integration tests involving nginz,
+    # nginz and brig must share the same zauth public/private keys
```

Comment:
is there an easy way to check this, and turn this comment into an error if it's violated?

Reply:
If it's violated, tests will fail with an "Unauthorized" 401 from nginz. What do you mean exactly with "turn into an error" ? Normally, nginz and brig don't run on the same machine, they run on different VMs or on different pods, so I'm not sure there can be a good way to detect this precise condition. Any ideas?

Reply:
hm, good point.

i suppose you could write a test that does nothing but try to get a 200 from nginz the easiest way possible, and if it gets a 401, the error has "make sure keys match, or something" in it.

You decide whether it's worth it.